x11-misc/screenkey/metadata.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+    <maintainer type="person">
+        <email>xgqt@protonmail.com</email>
+        <name>Maciej Barć</name>
+    </maintainer>
+    <maintainer type="project">
```

Comment:
Just add myself here (sultan@gentoo.org), no need for proxy-maint project.

Reply:
Without me?

Reply:
No, only remove the project and add myself instead. I will push the PRs for you.